### ISSUE TITLE
Try shorten Python filenames harder

### ISSUE THREAD

**Issue by aparamon** [benfred/py-spy#75]
Despite the intention:
https://github.com/benfred/py-spy/blob/0eba9ffc30c6472d08b8612d11e4a9d142f93021/src/python_spy.rs#L162-L181
some boilerplate is not removed from Python filenames, for example
````
_run_module_as_main (runpy.py);_run_code (runpy.py);<module> (local/lib/python3.6/dist-packages/pytest.py)
__call__ (/home/aparamon/.local/lib/python3.6/site-packages/pluggy/hooks.py)
````
It is proposed to support the above cases as well.

Also, it seems useful to left-strip current directory from Python filenames located there, e.g. keep just `tests.py` instead of verbose `/home/aparamon/projects/py/data-server/tests.py` when running from `/home/aparamon/projects/py/data-server` (and `data-server/tests.py` when running from `/home/aparamon/projects/py`).
````

**Comment by benfred** [comment]
Thanks for the report! I totally agree that this was lacking.

This is mostly handled here by this commit 3eb83dd1cc39c75fd2dee9b04d9b7cb13538fd03 - basically it's looking for ```__init__.py``` files in the directory and only showing directories containing those files (so only showing the directory if the file is in a python package or subpackage - but leaving out otherwise).  I'm hoping that this means we don't need to incorporate the CWD, since it's a little hard to get properly from the remote process on windows (notes here: https://github.com/benfred/py-spy/blob/c47d9acd62b05199a723e61e74931415ebd94b6f/remoteprocess/src/windows/mod.rs#L69-L83)

This will be in the next release - but if you want to test it out you can install the a pre-release version here ```pip install py-spy==0.2.0.dev0``` (also has initial version of code for profiling native/cython extensions)

**Comment by benfred** [comment]
fix is in v0.1.11
